introc/introc.tex

```diff
@@ -1355,35 +1361,35 @@ \subsubsection{String constants are constant}
 strcpy(ptr, "World"); // OK because now ptr is pointing to mutable memory (the array)
 \end{lstlisting}
 
-What to take away from this is that pointers * can point to any type of memory while the char arrays mentioned earlier will always refer to the same piece of memory.
+What one must take away from this, is that, pointers can point to any type of memory while the char arrays mentioned earlier will always refer to the same piece of memory.
```

Comment:
We should probably just rewrite this sentence.

Reply:
Done.